anchor-reference-server/src/main/java/org/stellar/anchor/reference/controller/Sep24InteractiveController.java

```diff
@@ -0,0 +1,29 @@
+package org.stellar.anchor.reference.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.stellar.anchor.exception.SepValidationException;
+
+import javax.servlet.http.HttpServletRequest;
+
+/** The controller that implement the endpoints of the Sep24 interactive flow. */
+@Controller
+@CrossOrigin(origins = "*")
+@RequestMapping("/")
+public class Sep24InteractiveController {
+  @CrossOrigin(origins = "*")
+  @RequestMapping(
+      value = "/sep24/interactive",
+      method = {RequestMethod.GET})
+  @ResponseBody
+  public String interactive(HttpServletRequest request) throws SepValidationException {
+    String operation = request.getParameter("operation");
+    if (operation == null) throw new SepValidationException("Missing [operation] parameter.");
+    if (operation.equals("withdraw")) return "The sep24 interactive WITHDRAW starts here.";
+    else if (operation.equals("deposit")) return "The sep24 interactive DEPOSIT starts here.";
+    else return "Undefined operation " + operation;
+  }
+}
```

Comment:
🤔  I don't quite understand this controller... what is it used for?

Reply:
This is the starting point of the SEP-24 interactive flow.

Reply:
Let me check if I understood: this is a piece of the reference server that the anchor would need to implement, right?

This is not part of the [SEP-24](https://stellar.org/protocol/sep-24) design nor the callback API design though, is it?